.dockerignore

```
@@ -0,0 +1,4 @@
+Dockerfile
+docker-compose.yml
+deployment/
+.*
```

Nice, we might also want to add any caching files, e.g.:
```suggestion
.*
**/__pycache__
**/.pytest_cache
```
(Note that `dockerignore` doesn't use _precisely_ the same syntax as `gitignore`, so we need to specify that those directories should be ignored regardless of arbitrary nesting)